packages/yarnpkg-core/sources/Configuration.ts

```diff
@@ -1293,6 +1301,11 @@ export class Configuration {
         throw error;
       }
 
+      if (key === `enableStrictSettings` && source !== `<environment>`) {
+        strict = parsed as boolean;
+        continue;
+      }
```

Comment:
This breaks `yarn config get`:
![image](https://user-images.githubusercontent.com/32596136/129185804-e84ee978-5b5d-4383-94a1-0ca624a3d682.png)

Reply:
This one is a bit weird but expected; since it only affects the file it's in, this setting shouldn't affect `config get`. Perhaps `config set` should throw an error though 🤔